src/commands/moderation/blacklist.ts

```diff
@@ -0,0 +1,127 @@
+
+/*
+ * Command for alerting the administrators about someone to blacklist
+ */
+
+const name: string = "blacklist";
+
+export const command: Command = {
+    // Command headers
+    data: <SlashCommandBuilder> new SlashCommandBuilder()
+        .setName(name)
+        .setDescription("Alert the Sushi admins about a potentially dangerous individual")
+        .setDMPermission(false)
+        .setDefaultMemberPermissions(PermissionsBitField.Flags.Administrator)
+        .addUserOption(o => 
+            o.setName("user")
+                .setDescription("User to report")
+                .setRequired(true)
+        )
+        .addStringOption(o =>
+            o.setName("reason")
+                .setDescription("Why should this user be blacklisted? Provide a short paragraph")
+                .setRequired(true)
+        ),
+
+    // Command execution
+    async execute(ctx: ChatInputCommandInteraction): Promise<void> {
+        // The user to blacklist
+        const user: User = ctx.options.getUser("user");
+        const uid: string = user.id;
+
+        // The reason provided by the mod/admin
+        const reason: string = ctx.options.getString("reason");
+
+        // The server this command was executed in
+        const server: Guild = ctx.guild;
+        // The user who used this command
+        const source: User = ctx.user;
+        // The channel to log the report in
+        const userReports: TextChannel = getUserReportsChannel();
+
+        await ctx.deferReply({ ephemeral: true });
+
+        const bl: BlacklistT = await Blacklist.findById(process.env.BLACKLIST_ID);
+
+        // Should ideally not be null but in the off-chance,
+        // recreate it
+        if (!bl) {
+            await createBlacklist();
+        }
+
+        const users: string[] = bl.users;
+
+        if (users.includes(uid)) {
+            await ctx.followUp({ content: "The specified user is already blacklisted", ephemeral: true });
+            return;
+        }
+
+        // Log the report in the user reports channel
+        const reportMsg: Message = await userReports.send(
+            "# USER REPORT\n" +
+            `**Server:** ${server.name}\n` +
+            `**Sent by:** <@${source.id}>\n` +
+            `**Target:** <@${user.id}>\n` +
+            `**Reason:** ${reason}\n` +
+            "--------------------"
+        );
+
+        // Add reactions for the admins to use
+        // Yellow for process started and green for process finished
+        await reportMsg.react("🟡");
+        await reportMsg.react("🟢");
+
+        await ctx.followUp({
+            content: "A report has been sent to the administrators, they will contact you as soon as possible. " +
+                     "Make sure that you have a sound argument and evidence of any potentially dangerous actions!\n" +
+                     `In the meantime, ban **${user.username}** from this server if you haven't already`,
+            ephemeral: true
+        });
```

Comment:
A report is sent to the admin server, informing the administrators about potentially malicious users